### ISSUE TITLE
Limit Cell contents to maximum of 32767 characters

### ISSUE THREAD

**Issue by PeteWitt** [dfinke/ImportExcel#1422]
Issue: wishing to store a (long) list of Windows Shares info to a single cell, discovered Excel reports 'found a problem, recover...' then  'Repaired Records: String properties from /xl/sharedStrings.xml part (Strings)' - though the XSLX is successfully opened.
Confirmed on ImportExcel v4.0.9 and v7.8.4  

Work around: I truncate the string, adding an ellipsis / continuation identifier, like
Set-Variable ExcelMaxCellChars -Value ([int16]32767) -Option Constant
amend string to $(($CellContents.Substring(0,[math]::min($ExcelMaxCellChars-3,$CellContents)))..."

Excel limits reference: https://support.microsoft.com/en-us/office/excel-specifications-and-limits-1672b34d-7043-467e-8e27-269d656771c3?ui=en-us&rs=en-us&ad=us#ID0EBABAAA=Excel_2016-2013

**Comment by PeteWitt** [comment]
Hello,

I had only tried Export-Excel. To be clear, the function doesn't error -
though it does produce an XLXS file somewhat incompatible with Excel.

Further testing shows Import-Excel is able to read such a file (containing
cell content > 32767 chars) without issue.

The same file causes Excel to throw a warning 'we found a problem ....
recover as much as we can?'. After responding Yes, 'Excel was able to open
the file by repairing or removing the unreadable content.' is reported and
we find the repaired XLSX's cell contents has been truncated to 32767 chars.

Regards,
Pete.
